bitnami/metallb/templates/deployment.yaml

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
The `---` is not necessary as this file only contains one resource.

Reply:
Fixed in the newer version.